public/js/clients/index.js

```diff
@@ -19,13 +19,17 @@ function debugPage(tab, actions) {
     if (isFirefox) {
       yield firefox.connectTab(tab.get("tab"));
       firefox.initPage(actions);
+      window.apiClient = firefox.getAPIClient();
     } else {
       yield chrome.connectTab(tab.get("tab"));
       chrome.initPage(actions);
+      window.apiClient = chrome.getAPIClient();
```

Comment:
I don't feel strongly about it, but since we do it for the store too, it might be nice to have a function like `getDebugObjects` or something that you can destructure at the console (`const { store } = getDebugObjects()`) so that they are all grouped in one place. Or maybe that function sticks it on window itself, so you just call `injectDebugObjects()`. We can do this later.